proposals/052-cryptography-leg-1.md

```diff
@@ -0,0 +1,393 @@
+
+The following are goals, but may not be complete by this proposal:
+
+- To provide a suitable alternative interface / backend (as an option) for `crypton`, `tls`, `x509`
+- To consolidate Haskell cryptography into a consistent, unified, user-friendly system
+- To enable the safe use and implementation of advanced cryptographic data structures
+
+The following are not goals of this proposal:
+
+- Implementing a hierarchy of high-level cryptographic abstractions
+- Matching interfaces 1:1 with existing libraries / interfaces
+- Implementing every algorithm permutation possible
+
+## Timeline
+
+This timeline will focus on the immediate goal of publishing the following libraries:
+
+- botan-bindings
+- botan-low
+- botan
+
+This proposal has a defined scope and set of deliverables, and will require substantial time to complete.
+
+### Expected project duration
+
+We estimate that it will take 3 months for this project, to achieve our goal of developing and publishing these libraries. This estimate is based on the significant amount of work already completed, the length of time in which it was accomplished, and the remaining amount of work.
+
+Several work items (strictness, memory safety, C++ shims for missing FFI functionality) are difficult to estimate, and so while time has been allocated for them in the estimate, their delivery is considered optional within the proposed timeline in order to ensure completion of the immediate goals. If necessary, the project may be extended at the discretion of the Haskell Foundation in order to ensure their completion.
+
+### Intermediate deliverables
+
+The following are intermediate deliverables intended to keep the community apprised of any progress that is made, and any challenges that arise, and to seek community feedback:
+
+- Frequent (2-3x weekly) activity update on devlog
+- Weekly major update that includes github push
+- Monthly report that summarizes what specific items have been worked on, what items will be worked on, and what challenges have arisen, if any.
+
+### Deliverables
+
+The following are the intended deliverables of this proposal:
+
+- `botan-bindings`
+- `botan-low`
+- `botan`
+
+Each library should be beta- or / production-ready, and submitted to hackage as a candidate or accepted package. 
+
+Each library should have:
+
+- Installation instructions for the Botan library
+- Documentation
+- Tutorials and examples
+- Tracked issues
+- Unit tests
+- CI tests for Linux, MacOS, and Windows
+
+The official repo will be owned by the [Haskell Cryptography Group ](https://github.com/haskell-cryptography) and maintained by myself as a member of the group. Another member of the Haskell Cryptography Group, or a member of the Haskell Foundation, will be selected to hold backup keys and permissions to the website, server, and official repositories as needed, in order to give project members access and to reduce any conflicts about ownership or maintenance.
```

Comment:
I cannot be sure and I appreciate that it's a delicate matter, but if I read your intention right, I think it would be better served by explicitly granting ownership to HF and maintainership to HCG.

Reply:
That might be more appropriate, and aligns with my intent.